pennylane/devices/default_gaussian.py

```diff
@@ -888,3 +888,13 @@ def operations(self):
     @property
     def observables(self):
         return set(self._observable_map.keys())
+
+    def batch_execute(self, circuits):
+        if not qml.active_return():
+            return super().batch_execute(circuits)
+        for circuit in circuits:
+            if len(circuit.measurements) > 1:
+                raise qml.QuantumFunctionError("Default gaussian only support single measurements.")
```

Comment:
What happens if such a circuit passed to execute instead of batch_execute?

Reply:
Do you want me to add the error in execute as well?

Reply:
I changed the execute to have it in. Codefactor is omplaining but it is not a problem as pylint is passing.